lnrpc/rpc.proto

```diff
@@ -988,11 +1011,17 @@ message ChannelAcceptResponse {
 
 message ChannelPoint {
     oneof funding_txid {
-        /// Txid of the funding transaction
+        /**
+        Txid of the funding transaction.  When using REST, this field must be
+        encoded as base64.
+        */
         bytes funding_txid_bytes = 1 [json_name = "funding_txid_bytes"];
 
-        /// Hex-encoded string representing the funding transaction
-        string funding_txid_str = 2 [json_name = "funding_txid_str"];
+        /**
+        Hex-encoded string representing the funding transaction. Deprecated now
+        that the REST gateway supports base64 encoding of bytes fields.
+        */
+        string funding_txid_str = 2 [json_name = "funding_txid_str", deprecated = true];
```

Comment:
This makes the whole `oneof` redundant. But maybe that removal should be done in a second step?

Reply:
I removed the deprecation from this field again. This is the case @Roasbeef mentioned where it might be useful for the users to be able to just pass in the string representation of a TXID that they get from a previous output and don't need to byte-reverse it before feeding it into the RPC/REST call. So the `oneof` can also remain.